packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/pages/api/editing/config.ts

```diff
@@ -10,6 +10,7 @@ import metadata from 'temp/metadata.json';
 const handler = new EditingConfigMiddleware({
   components,
   metadata,
+  enableEditingMetadata: true,
```

Comment:
This will make config endpoint to always be called with in 'metadata' edit mode. My understanding is Pages should be able to call it with both 'Chromes' and 'Metadata' modes. Is that right? If so, there should be a way to alternate between true and false values here.